.gitignore

```diff
@@ -31,6 +31,7 @@ test/datalog_tests/*.dump
 test/datalog_tests/*.c
 test/datalog_tests/*_ddlog
 test/datalog_tests/debug.log
+test/datalog_tests/prof.profile
```

Comment:
This should go in `.git/info/exclude` instead of the committed gitignore

Reply:
```
[cargo-make] INFO - Running Task: download-data
downloading '' to 'data/' 
[cargo-make] ERROR - Error while running duckscript: Source: Unknown Line: 10 - Invalid URL:  provided.
[cargo-make] WARN - Build Failed.
```
There isn't really a task download-data, it is a task invoked by other tasks.

Reply:
Why should it be in .git/info/exclude? I assume no one wants to commit that file by mistake, as I did.

Reply:
The file existing isn't part of the project, it's an artifact of your particular dev flow

Reply:
It was generated when running some tests with their default flags, I didn't do anything special for that.